src/App.js

```diff
@@ -2,77 +2,50 @@ import React, { Component } from 'react';
 import './App.css';
 
 class App extends Component {
-  _isMounted = false;
-
   state = {
-    source: 'https://i.imgur.com/VYUdUNJ.png',
+    source: '',
     error: null,
+    loading: true,
   };
 
-  fetchCat() {
-    fetch("https://aws.random.cat/meow").then(res=>res.json()).then(
-      (result) => {
-        this.setState({
-          source: result.file
-        });
-    },
-    (error) => {
-      this.setState({
-        error
-      });
-    }
-    )
-  }
-
   componentDidMount() {
-    this._isMounted = true;
-    const { source } = this.state;
-    this.fetchCat(source);
-
+    this.fetchCat();
   }
 
-  componentWillUnmount() {
-    this._isMounted = false;
+  fetchCat = async () => {
+    try {
+      const result = await fetch("https://aws.random.cat/meow");
+      const image = result.json();
+      this.setState({ source: image.file, loading: false });
+    } catch (e) {
+      this.setState({ error: e.message, loading: false });
+    }
   }
 
   render() {
-    const { error, source } = this.state;
-    if (error) {
-      return <div>Error: {error.message}</div>;
-    }
-    return (
+    const { error, loading, source } = this.state;
+    return loading ? (
+      <p>Loading...</p>
+    ) : error ? (
+      <div>Error: {error}</div>
+    ) : (
       <div className="App">
         <div className="container">
           <div className="row">
             <div className="col-1"></div>
             <div className="col-10">
-              <Image 
-                src={source}
-                className="img-fluid"
-                alt="cat picture"
-              />
+              <img src={source} className="img-fluid" alt="cat picture" />
             </div>
             <div className="col-1"></div>
           </div>
-          <Button
-            text="Get Cat"
-            type="button"
-            className="btn btn-light"
-            onClick={() => this.fetchCat()}
-          />
```

Comment:
Comments are the same as above.

Only additional comment is that writing `() => this.fetchCat()` is the same as writing `this.fetchCat`. The only reason you would need the first example is when you want to use parameters. For example `() => this.fetchCat(catId, catBreed)` is correct and `this.fetchCat(catId, catBreed)` will throw errors in your app